### PR TITLE
Inline load scripts for integration tests

### PR DESCRIPTION
**Pull request by marcusoffesson**
Partial integration test cleanup: inline load scripts instead of reading them from file system.

### CODE REVIEW

File: test/integration/sugar-mixins.spec.js [hunk 1]
```diff
@@ -11,13 +12,11 @@ describe('Sugar mixins', () => {
     config.mixins = [{
       types: 'Global',
       extend: {
-        createAppAndLoad(appName, scriptName) {
+        createAppAndLoad(appName, loadScript) {
           return qixGlobal.createApp(appName).then(appInfo =>
```
Comment (peol) [inline test/integration/sugar-mixins.spec.js]: Shouldn't we switch to using session apps, or you mean to do that in another PR?

Comment (marcusoffesson) [inline test/integration/sugar-mixins.spec.js]: I'm doing that as well, but I want to keep the PRs as manageable (small and isolated) as possible. Consider it a courtesy to all of you reviewers!  :)

**Comment by marcusoffesson** [comment]
@axelssonHakan Sure, more script files can easily be added. The reason for doing this change is that it simplifies the promise chains quite a lot, and it makes the tests more robust not having to rely on reading from file system, etc.
